.github/workflows/markdown.yml

```diff
@@ -0,0 +1,20 @@
+name: Markdown lint
```

Comment:
Nit: was this meant to be `Markdownlint`, no space? My first instinct is that the space is a typo, but I guess it makes sense either way...? 😄